frontend/src/components/assessments/student-experience/question-elements/ShortAnswer.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Input } from "@chakra-ui/react";
+
+import { stringToNumberArray } from "../../../../utils/StudentUtils";
+
+import useAnswerState from "./useAnswerState";
+
+interface ShortAnswersProps {
+  answerElementIndex: number;
+}
+
+const ShortAnswer = ({
+  answerElementIndex,
+}: ShortAnswersProps): React.ReactElement => {
+  const { currentAnswer, updateAnswer } = useAnswerState(answerElementIndex);
+
+  return (
+    <Input
+      borderColor="grey.300"
+      borderRadius="8px"
+      focusBorderColor="grey.300"
+      onChange={(e) => updateAnswer(stringToNumberArray(e.target.value))}
+      placeholder="Write your answer here"
+      type="number"
+      value={currentAnswer[0] ?? ""}
+      variant="outline"
+      width="34%"
```

Comment:
Will this behave correctly on different screen sizes? (e.g. large ones)

Reply:
Tested this on 1920x1080 and looks alright to me? Let me know if i'm misunderstanding the comment :D Or is the idea that we want the short answer bar to be less wide?

<img width="1175" alt="Screenshot 2023-05-19 at 5 25 10 PM" src="https://github.com/uwblueprint/jump-math/assets/41273486/d6a02957-86b8-4fa4-a7e3-7136605f5f32">

Reply:
I was thinking like super wide but it's probably an edge case we don't really need to worry about much